src/ManifestStrategy/WireNavigationManifestStrategy.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\ManifestStrategy;
+
+use SimpleXMLElement;
+use SprykerSdk\Integrator\Dependency\Console\InputOutputInterface;
+use SprykerSdk\Integrator\Exception\UnexpectedNavigationXmlStructureException;
+
+class WireNavigationManifestStrategy extends AbstractManifestStrategy
+{
+    /**
+     * @var string
+     */
+    protected const TARGET_NAVIGATION_FILE = 'config/Zed/navigation.xml';
+
+    /**
+     * @var string
+     */
+    protected const KEY_NAVIGATIONS_CONFIGURATION = 'navigations';
+
+    /**
+     * @var string
+     */
+    protected const KEY_NAVIGATION_POSITION_AFTER = 'after';
+
+    /**
+     * @var string
+     */
+    protected const KEY_NAVIGATION_POSITION_BEFORE = 'before';
+
+    /**
+     * @return string
+     */
+    public function getType(): string
+    {
+        return 'wire-navigation';
+    }
+
+    /**
+     * @param array<mixed> $manifest
+     * @param string $moduleName
+     * @param \SprykerSdk\Integrator\Dependency\Console\InputOutputInterface $inputOutput
+     * @param bool $isDry
+     *
+     * @return bool
+     */
+    public function apply(array $manifest, string $moduleName, InputOutputInterface $inputOutput, bool $isDry): bool
+    {
+        if (!file_exists($this->getNavigationSourceFilePath())) {
+            $inputOutput->writeln(sprintf(
+                'The project doesn\'n have the navigation source file: %s.',
+                static::TARGET_NAVIGATION_FILE,
+            ), InputOutputInterface::DEBUG);
+
+            return false;
+        }
+
+        try {
+            $navigationConfiguration = $this->getNavigationConfiguration();
+        } catch (UnexpectedNavigationXmlStructureException $unexpectedNavigationXmlStructureException) {
+            return false;
```

Comment:
Do we need to log this exception?

Reply:
I just created this exception in order to control behavior from another method. I guess we should not log it.

Reply:
I would also not log it.